generators/app/templates/8/docs/_build.md

```diff
@@ -13,5 +13,5 @@ Builds are created in the `web` directory.
 ## Executing a build
 
 ```bash
-npm run build-site
```

Comment:
I also included composer install cleanup into this PR. So, it means to remove that command in favor of simply `composer install`